src/viur/toolkit/db.py

```diff
@@ -159,15 +159,18 @@ def transaction() -> skeleton.SkeletonInstance | db.Entity:
         return obj
 
     # Retry loop
```

Comment:
I would prefer a less indented and negated version:
```suggestion
    if db.IsInTransaction:
        return transaction()
    # Retry loop
```